app/models/ems_event.rb

```diff
@@ -227,6 +227,7 @@ def get_target(target_type)
 
     target_type = "src_vm_or_template"  if target_type == "src_vm"
     target_type = "dest_vm_or_template" if target_type == "dest_vm"
+    target_type = "middleware_server"   if event.event_type == "hawkular_event"
```

Comment:
> Yes, I think we could need a more specific event_type for alert-related events.

I am OK if this is done in a follow-on PR it should just be noted that until we change to an internal event_type the detail timeline view will, I think, show these alerting events.

> Middleware_server is the appropiate target_type for this end to end

I am wondering whether a single target type, like 'Middleware_Alerting' may be possible, with the entity type being determined by inspecting the event to see which middleware_entity is set.  We do something analogous on the timeline view where we create a link to the entity based on which entity is set on the event_streams record.